backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/spreadsheet/excel/JdbcExcelExporter.java

```diff
@@ -81,6 +83,7 @@ private JdbcExcelExporter(
 		this.translateHeaders = translateHeaders != null ? translateHeaders : true;
 		this.resultFile = resultFile;
 		this.noDataAddedYet = true;
+		this.fileName = fileName;
```

Comment:
this.filenamePrefix = StringUtils.trimBlankToOptional(filenamePrefix).orElse('Report');

Reply:
done